website/docs/docs/getting-started/01-Installation.md

```diff
@@ -108,3 +118,69 @@ helm uninstall kro -n kro
 Keep in mind that this command will remove all kro resources from your cluster,
 except for the ResourceGraphDefinition CRD and any other custom resources you may have
 created.
+
+## Installation with kubectl
+
+### Installation Steps
+
+#### Install kro using Kubectl
+
+Select the variant you want (see available variants in [prerequisites](#prerequisites))
```

Comment:
do we have an overview of these variants somewhere? if not, maybe we should include them as part of the release (where you started including them) or listing them here